dcrec/secp256k1/curve.go

```diff
@@ -149,6 +149,18 @@ func (p *JacobianPoint) ToAffine() {
 	p.Y.Normalize()
 }
 
+// EquivalentNonConst returns whether or not two Jacobian points represent the
+// same affine point in *non-constant* time.
+func (p *JacobianPoint) EquivalentNonConst(other *JacobianPoint) bool {
+	// Use the group law that a point minus itself is the point at infinity to
```

Comment:
While this statement is true, it might not be obvious for those who aren't super familiar with group laws and their implications, so I wonder if rather than just asserting the implication this would be clearer as:

```Go
       // Since the point at infinity is the identity element for the group, note  
       // that P = P + ∞ trivially implies that P - P = ∞.
       //
       // Use that fact to determine if the points represent the same affine point.
```

Reply:
it read fine as is before, but this is ok too.

Reply:
Updated.